recipients-resolver/src/main/java/com/redhat/cloud/notifications/recipients/resolver/FetchUsersFromExternalServices.java

```diff
@@ -251,9 +251,13 @@ List<User> transformItUserToUser(List<ITUserResponse> itUserResponses) {
             user.setId(itUserResponse.id);
             user.setUsername(itUserResponse.authentications.get(0).principal);
 
-            for (Email email : itUserResponse.accountRelationships.get(0).emails) {
-                if (email != null && email.isPrimary != null && email.isPrimary) {
-                    user.setEmail(email.address);
+            if (itUserResponse.accountRelationships.get(0).emails == null) {
+                Log.infof("Skipping user account id: %s with username: %s because it don't have any email addresses", user.getId(), user.getUsername());
```

Comment:
```suggestion
                Log.infof("Skipping user account id: %s with username: %s because it doesn't have any email addresses", user.getId(), user.getUsername());
```